contracts/governance/Staking/Checkpoints.sol

```diff
@@ -44,7 +44,7 @@ contract Checkpoints is StakingStorage, SafeMath96 {
     );
 
     /// @notice An event emitted when vesting tokens get withdrawn.
-    event VestingTokensWithdrawn(address vesting, address receiver);
+    event VestingTokensWithdrawn(address vesting, address receiver, uint256 startFrom);
```

Comment:
this event is not emitted anywhere
pls remove

Reply:
fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/d448c6258afce4aaede3f46f568b92fea9dc2e9d)